### ISSUE TITLE
 Add inCollection-Element to Alma-Transformation

### ISSUE THREAD

**Issue by TobiasNx** [hbz/lobid-resources#1114]
- [x] "inCollection.[].id"
- [x] "inCollection.[].label"
- [x] "inCollection.[].type.[]"

Add different Collections:

- [x] ZDB (constant)
- [x] EZB (constant)
- [x] nwbib (constant)
- [x] eResource package: value differs as 912  .a, thhere can be multiple objects with "eResource package"
- [ ] edoweb
- [ ] rpb
- [ ] publisso

**Comment by TobiasNx** [comment]
The inCollection type is still making trouble.

**Comment by TobiasNx** [comment]
I need the variable "idzdb" that is created in #1114. So #1113 should be finished first.

**Comment by TobiasNx** [comment]
edoweb, rpb, publisso need to be added later, when we find testdata for that

**Comment by TobiasNx** [comment]
New ticket for missing collections: #1123

**Comment by TobiasNx** [comment]
Test for publisso "http://lobid.org/resources/HT019838800#!"

**Comment by TobiasNx** [comment]
Test for edoweb: "http://lobid.org/resources/HT019440025#!"
